### PR TITLE
Pep8

### PR DESCRIPTION
**Pull request by BRJMM**

### CODE REVIEW

File: eyes-servo-tes.py [hunk 1]
```diff
@@ -36,246 +25,213 @@
...
 ]
 
-#util vars
+# util vars
 wait_ms = 50
 wait_s = 3
 
-#prints a eye based on a emotion
+
 def print_conEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: Functions are all lower case

File: eyes-servo-tes.py [hunk 2]
```diff
@@ -36,246 +25,213 @@
...
     time.sleep(wait_s)
-    #to origin
+    # to origin
     right_ear.write(RIGHT_EAR_ORIG)
     left_ear.write(LEFT_EAR_ORIG)
-    #h.write(0.415)
-    #e.write(0.445)
+
 
 def print_surEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: Lower case plase

File: eyes-servo-tes.py [hunk 3]
```diff
@@ -36,246 +25,213 @@
...
-    #jaws.write(JAWS_ORIG)
     right_ear.write(RIGHT_EAR_ORIG)
     left_ear.write(LEFT_EAR_ORIG)
-    #g.write(0.47)
-    #h.write(0.415)
-    #e.write(0.445)
-    #f.write(0.435)
+
 
 def print_sadEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py [hunk 4]
```diff
@@ -36,246 +25,213 @@
...
-    #Mouth
-    #jaws.write(0.3)
-    #Ears
-    #to origin
-    #jaws.write(JAWS_ORIG)
+    # TODO: put servo movements here!!!
     time.sleep(wait_s)
 
+
 def print_hapEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py [hunk 5]
```diff
@@ -36,246 +25,213 @@
...
     time.sleep(wait_s)
-    #back to origin
-    #jaws.write(JAWS_ORIG)
     right_ear.write(RIGHT_EAR_ORIG)
     left_ear.write(LEFT_EAR_ORIG)
-    #g.write(0.47)#48
-    #f.write(0.435)#45
+
 
 def print_calEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: lower case please

File: eyes-servo-tes.py [hunk 6]
```diff
@@ -36,246 +25,213 @@
...
-           j = 0
+            j = 0
         else:
-           j += 1
+            j += 1
     strip.show()
     time.sleep(wait_s)
 
+
 def print_eneEye(strip):
```
Comment (degv364) [inline eyes-servo-tes.py]: lowercase please

File: eyes-servo-tes.py [hunk 7]
```diff
@@ -36,246 +25,213 @@
...
     for i in range(strip.numPixels()):
         strip.setPixelColor(i, ene_eye[i])
     strip.show()
     time.sleep(wait_s)
 
-#puts a color on every pixel 
+
 def colorWipe(strip, color, wait_ms=50):
```
Comment (degv364) [inline eyes-servo-tes.py]: lowercase please

**Review by degv364 — CHANGES_REQUESTED** [review]
According to PEP8 function names should be in lowercase.

**Comment by BRJMM** [comment]
changes made

**Review by degv364 — APPROVED** [review]

**Comment by degv364** [comment]
Looks good to me. But lets wait for another reviewer to approve.

**Comment by BRJMM** [comment]
I will wait the another approve to continue

**Comment by jprltsnz** [comment]
Changed merged, thanks!

**Pull request merged**